static/sass/styles.scss

```diff
@@ -30,7 +30,7 @@ $color-text-orange: #d54110;
 @include vf-p-icons;
 @include vf-p-icon-fullscreen;
 @include vf-p-image;
-@include vf-p-label;
+@include vf-p-status-label;
```

Comment:
Thanks, we gonna fix the broken removed mixin in next release, but updating it now is even better.